contrib/dev/Vagrantfile

```diff
@@ -135,8 +139,8 @@ def dev_synced_folders(override, type)
 end
 
 Vagrant.configure(2) do |config|
-  config.vm.define "dev" do |dev|
-    dev.vm.hostname = "dev"
+  config.vm.define HOSTNAME do |dev|
```

Comment:
The name of the VM should not change and stay "dev"

Reply:
left hostname "dev" but I allow to change the VM Name